RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
It does, but it looks like there's a bug in MSBuild's `ResolveAssemblyReferences` task that prevents it from detecting the versions conflict (and generating the required binding redirect): https://github.com/dotnet/corefx/issues/15570#issuecomment-286810713.